src/main/java/eu/mikroskeem/worldeditcui/gui/SettingsEntry.java

```diff
@@ -29,11 +29,23 @@ public void render(int index, int y, int x, int rowWidth, int rowHeight, int mou
         RenderSystem.color4f(1.0f, 1.0f, 1.0f, 1.0f);
         String name = keyword;
         String trimmedName = name;
-        int maxNameWidth = rowWidth - 32 - 3;
+        int maxNameWidth = rowWidth / 2 - 40;
         if (this.client.textRenderer.getStringWidth(name) > maxNameWidth) {
-            trimmedName = this.client.textRenderer.trimToWidth(name, maxNameWidth - this.client.textRenderer.getStringWidth("...")) + "...";
+            trimmedName = this.client.textRenderer.trimToWidth(name, maxNameWidth);
+            int lengthTemp = 0; String temp1String = ""; String temp2String = "";
+            for(String text : name.split(" ")) {
+                if((text.length() + lengthTemp) <= trimmedName.length()) {
+                    lengthTemp += text.length();
+                    temp1String += text + " ";
+                } else {
+                    temp2String += text + " ";
+                }
+            }
+            this.client.textRenderer.draw(temp1String, x + 50, y - 4, 0xFFFFFF);
+            this.client.textRenderer.draw(temp2String, x + 50, y + 6, 0xFFFFFF);
```

Comment:
What does this section do, split a long line up to two lines? Can't this be implemented in a better way? I see a possibility to use StringBuilder here.

Reply:
Yeah, its a section that maybe is too messy, it split the string if its too long, depending on the acreen width